drivers/hmis/app/graphql/types/forms/enable_when.rb

```diff
@@ -8,6 +8,7 @@
 
 module Types
   class Forms::EnableWhen < Types::BaseObject
+    skip_activity_log
```

Comment:
> Types::Forms::AutofillValue also needed this to work 

Thanks! I'll adjust AutofillValue. 

> maybe there should be a base type for the form types that has it?

Are you thinking we'll need a bundle of additional behaviors on these form types? If not then I prefer sticking with the class attr (`skip_activity_log`) to control behavior; we can extract it to a mixin or something if needed in the future.

Reply:
No, I think that's fine. I was just thinking they are special since they're not backed by records, but I don't think there is any additional behavior we need for them yet.